src/tests/VestRewardsDistribution-integration.t.sol

```diff
@@ -0,0 +1,205 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Copyright (C) 2021 Dai Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity 0.8.13;
+
+import "forge-std/Test.sol";
+import "./DssCronBase.t.sol";
+
+import {VestRewardsDistributionJob} from "../VestRewardsDistributionJob.sol";
+
+interface VestedRewardsDistributionLike {
+    function dssVest() external view returns (address);
+    function vestId() external view returns (uint256);
+}
+
+interface DssVestLike {
+    function unpaid(uint256 _id) external view returns (uint256 amt);
+}
+
+contract VestRewardsDistributionJobIntegrationTest is DssCronBaseTest {
```

Comment:
Probably worth adding a comment that this test is expected to run in fork mode against the following RPC URL:
```
https://virtual.mainnet.rpc.tenderly.co/470dbf59-a384-4e77-974c-9430acb2fccb
```